pages/dkp/kaptain/2.0.0/operations/mysql-cluster/index.md

```diff
@@ -0,0 +1,274 @@
+---
+layout: layout.pug
+navigationTitle: MySQL Cluster
+title: Scalability and operations for the Kaptain MySQL Cluster
+menuWeight: 70
+excerpt: Scalability and operations for the Kaptain MySQL Cluster
+beta: false
+enterprise: false
+---
+
+Learn how to scale and backup/restore the MySQL cluster with Kaptain. Kaptain provides a consolidated
+MySQL cluster with primary-primary replication for storage of the Pipelines execution history and artifacts,
+and Katib experiment results. 
+
+## Prerequisites
+
+-   You already provisioned a Konvoy cluster using at least `v1.7.0`.
+
+## Overview
+By default, the MySQL cluster is based on Percona XtraDB, and consists of 3 database nodes and 2 proxy nodes. The database nodes run MySQL daemons and store the data, and the proxy nodes run HAProxy and expose the database to the clients via a Kubernetes Service.
+
+The database nodes form a primary-primary database cluster.  This means there are no secondary nodes; i.e. all data is consistent across all the cluster nodes whenever a database operation completes successfully.
+
+Horizontally scaling the database nodes will result in more replicas of the data available for recovery in case of a failure.  However, it will not result in better performance because of the replication model.
```

Comment:
```suggestion
Horizontally scaling the database nodes will result in more replicas of the data available for recovery in case of a failure. However, it will not result in better performance because of the replication model.
```